script.js

```diff
@@ -0,0 +1,45 @@
+// Set variables for each respective move
+var x = "X"
+var o = "O"
+// Create functions to place X or O, taking in boxNumber to choose where to place
+function placeX(boxNumber) {
+	document.getElementById(boxNumber).innerHTML = x;
+}
+function placeO(boxNumber) {
+	document.getElementById(boxNumber).innerHTML = o;
+}
+
+// Set variables to easily select individual cells and determine which player's turn it is
+
+var cells = document.getElementsByTagName("TD");
+var playerTurn = document.getElementsByClassName("player-turn")[0];
+
+
+var moveCounter = 1;
+playerTurn.innerHTML = "X's turn!"
+function makeMove(boxNumber) {
+	moveCounter++;
+	if(cells[boxNumber-1].innerHTML != "X" || "O") {
```

Comment:
This line has a subtle bug I believe, it should instead be:
```js
if((cells[boxNumber-1].innerHTML != "X") || (cells[boxNumber-1].innerHTML != "O"))
```
That is, you have to explicitly make the `!=` comparison twice, otherwise you're just asking if the cell text is "X", or if the string literal "O" is truthy (it always is!).

even better to write it like so (using a var to reduce duplication and improve readability):
```js
var currentText = cells[boxNumber-1].innerHTML;
if(currentText != "X" || currentText != "O")
//...
```